renovate.json

```diff
@@ -15,7 +15,12 @@
     {
       "matchPackageNames": ["phpstan/extension-installer"],
       "rangeStrategy": "replace"
-    }
+    },
+//    {
```

Comment:
@spawnia now the fixer runs in PRs, therefore it should not happen that something needs to be fixed in master.

Here I'd enable Renovate to automerge fixer version bumps.

Reply:
I would be fine with that, thanks for making this work.